### PR TITLE
Add help information about Solidus Compare

### PR DESCRIPTION
**Pull request by blocknotes**
- Update the documentation to add development information
- Add -h (or --help) option to show the help section in `solidus_compare`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Solidus Starter Frontend development information
+This document aims to give some extra information for developers that are going 
+to contribute to our `solidus_starter_frontend` component.
+
+## Solidus Compare tool
+`solidus_compare` is a tool that we created to keep track of the changes made to
+`solidus_frontend`, which we used as source project in the beginning.
```
Comment (kennyadsl) [inline docs/development.md:7]: Maybe we can link to it?

File: bin/solidus_compare [hunk 2]
```diff
@@ -14,6 +14,16 @@ CONFIG_PATH = 'config/solidus_compare.yml'
 @cmd_options = {}
 ARGV.each do |arg|
   case arg
+  when '-h', '--help'
```
Comment (kennyadsl) [inline bin/solidus_compare:17]: ❤️

**Review by kennyadsl — APPROVED** [review]
👍

File: docs/development.md (hunk 1 above)
Comment (blocknotes) [reply docs/development.md]: Good point.
Push forced a fixup to add the link to https://github.com/solidusio/solidus/tree/master/frontend

**Pull request merged**